### ISSUE TITLE
Too long lines (E501) fail to be reported when ending with an open triple-quoted string

### ISSUE THREAD

**Issue by dawagner** [PyCQA/pycodestyle#679]
I initialiy [reported this issue on flake8](https://gitlab.com/pycqa/flake8/issues/367) and got redirected here.

Too long lines (E501) fail to be reported when ending open with a triple-quoted string (aka longstring).

Here's an example:

`foo.py`:
```{python}
$ cat foo.py 
len("this line if 86 chars long and is not reported.....", "foo bar baz spam egg", """
        some other string""")

len("this line is 86 chars long and is reported.........", """foo bar baz spam egg""",
    "some other string")

```
```{sh}
$ flake8 foo.py 
foo.py:4:80: E501 line too long (86 > 79 characters)
```

**Comment by bdmbdsm** [comment]
I will take on this task if you do not mind it

**Comment by dawagner** [comment]
Please do 👍

**Comment by bdmbdsm** [comment]
Sorry, I became short on time - must concentrate on my new project. I will not work on this issue.

**Comment by r3m0t** [comment]
It's fixed already.

```commit cb17d2739696df66c5b1eed2de559b4d778836e2
Author: Anthony Sottile <asottile@umich.edu>
Date:   Wed Mar 15 08:37:40 2017 -0700

    Correctly report E501 when the first line of a docstring is too long
    
    Resolves #622
```

**Comment by sigmavirus24** [comment]
Thanks for noticing that @r3m0t

**Comment by manu2504** [comment]
Hello - the issue still persists with pycodestyle v2.8.0.

You may reproduce it with following snippet:
```python
no_flake8_e501_error = "My line longer than 80 chars xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" + '''
'''

flake8_e501_error = "My line longer than 80 chars xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
```

Running flake8 on this file results with:
> e501.py:4:81: E501 line too long (103 > 80 characters)

Here are my flake8 installed plugins:
```shell
$ poetry run flake8 --bug-report 
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.9.6",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.8.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.4.0"
    }
  ],
  "version": "4.0.1"
}
```

@r3m0t

**Comment by asottile** [comment]
this seems to be flake8 specific, I'll open an issue there: https://github.com/PyCQA/flake8/issues/1534
